docs/cloud-edge/modules/user-agent-filter.md

```diff
@@ -0,0 +1,127 @@
+# User Agent Filter
+
+---
+
+## Overview
+
+The User Agent Filter module allows you to `allow` or `deny` traffic based on the `User-Agent` header. You can define `allow` and `deny` regular expression rules that apply to the incoming `User-Agent` header.
```

Comment:
```suggestion
With the User Agent Filter module you can allow or deny traffic through regular expression-based rules that are matched against the incoming user-agent header, which contains information about the requesting browser or application, including its name, version, and operating system.

Since rules are processed at the ngrok Cloud Edge, you can bolster your application’s security by blocking unwanted or unauthorized requests from malicious actors before it ever reaches your network.
```